assets/styles/global.scss

```diff
@@ -578,3 +581,11 @@ a.has-text-grey {
 .has-text-inherit {
   color: inherit !important;
 }
+
+.min-width-0 {
+  min-width: 0;
+}
+
+.pointer-events-none {
+  pointer-events: none !important;
+}
```

Comment:
not needed?

Reply:
needed , had to add that , otherwise you could click on the checkbox without triggering the dropdown item select  
![CleanShot 2023-12-07 at 12 04 47@2x](https://github.com/kodadot/nft-gallery/assets/44554284/475ba294-8afb-46cc-99a0-b905311077b7)